pkgs/fc/agent/setup.py

```diff
@@ -10,14 +10,14 @@
 with open(path.join(here, 'README.rst'), encoding='utf-8') as f:
     long_description = f.read()
 
-
 test_deps = [
     'freezegun>=0.3',
     'responses',
     'pytest>=3',
-    'pytest-cov',
-]
-
+    'pytest-mock',
+    'pytest-structlog',
+    'pytest-subprocess',
+    'pytest-cov',]
```

Comment:
such formatting defeats the purpose of the trailing comma, which is to prevent merge conflicts
the ] should be on another line imo